flatpak/com.github.Alcaro.Flips.json

```diff
@@ -0,0 +1,37 @@
+{
+    "app-id" : "com.github.Alcaro.Flips",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.34",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "flips",
+    "finish-args" : [
+        /* X11 + XShm access */
```

Comment:
The lack of comments in JSON really is a pain. Flatpak allows YAML manifests too, which are newer (though are around since more than a year and hence pretty well supported), tend to be more compact, and more importantly allow comments. Maybe we should use them instead? The only reason to keep JSON is because GNOME Builder doesn't support YAML yet, but it will likely affect nobody.